llama_test.go

```diff
@@ -0,0 +1,60 @@
+package llama_test
+
+import (
+	"fmt"
+	"os"
+
+	. "github.com/go-skynet/go-llama.cpp"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("LLama binding", func() {
+	testModelPath := os.Getenv("TEST_MODEL")
+
+	Context("Declaration", func() {
+		It("fails with no model", func() {
+			model, err := New("not-existing")
+			Expect(err).To(HaveOccurred())
+			Expect(model).To(BeNil())
+		})
+	})
+	if testModelPath != "" {
```

Comment:
ginkgo has `Skip()` see https://github.com/go-skynet/LocalAI/blob/77e1ae3d707d94b819fe774ec886127fabadb335/api/api_test.go#L293 for example to skip tests